app/components/ranking.jsx

```diff
@@ -11,64 +12,127 @@ const selectedOption = (
   </>
 )
 
-const unselectedOption = <rect x="1" y="1" width="22" height="22" rx="11" stroke="#06335C" strokeWidth="2" />
+const unselectedOption = <rect x="1" y="1" width="22" height="22" rx="11" stroke="#5D5D5C" strokeWidth="2" />
```

Comment:
It's uncomfortable to have these hard coded color values in this file.  I know that's the way it was, but I have an idea to improve it. I wouldn't ask for this if there weren't other change to make, but since there are lets give this a try.

Please try defining the color in theme.js -- if it isn't already there, and then `import theme from theme.js` at the top and user theme.color.thatColor here.  That way all the colors can be centralized.   There are a few other places where there is a hard coded color value, please try this there too.